src/mlpack/tests/io_test.cpp

```diff
@@ -72,7 +70,8 @@ void AddRequiredCLIOptions()
  * Tests that CLI works as intended, namely that IO::Add propagates
  * successfully.
  */
-BOOST_AUTO_TEST_CASE(TestCLIAdd)
+TEST_CASE_METHOD(IOTestDestroyer, "TestCLIAdd",
+                "[IOTest]")
```

Comment:
```suggestion
TEST_CASE_METHOD(IOTestDestroyer, "TestCLIAdd", "[IOTest]")
```

Looks like this would fit into the same line.